### PR TITLE
Get GUI tests running again

### PR DESCRIPTION
**Pull request by hatched**
Setting up the GUI phantomjs tests again after the new makefile conversion.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2752/
Test FAILed.

**Comment by hatched** [comment]
#### To QA

To run the tests in phantomjs
`make test-js-phantom`

The tests should pass. If you have problems with phantomjs crashing then you'll have to visit the tests in chrome by visiting `{ip}:8888/test/index.html`

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2753/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2754/
Test PASSed.

File: jujugui/static/gui/src/test/assets/chai.js [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../../../node_modules/chai/chai.js
```
Comment (makyo) [inline jujugui/static/gui/src/test/assets/chai.js]: Thanks, GH, really useful

File: jujugui/static/gui/src/test/index.html [hunk 2]
```diff
@@ -71,16 +71,41 @@
...
+  <script>
+    var GlobalConfig = {
+      combine: true,
+      base: '/combo?/app/assets/javascripts/yui/',
+      comboBase: '/combo?',
+      maxURLLenght: 1300,
+      root: 'app/assets/javascripts/yui/',
+      groups: {
+        app: {
+            //combine: true,
```
Comment (makyo) [inline jujugui/static/gui/src/test/index.html]: Leftover?  If not, a comment as to why this is commented out might help

File: jujugui/static/gui/src/test/index.html [hunk 3]
```diff
@@ -186,8 +208,8 @@
   <script src="test_simulator.js"></script>
 
   <!-- FIXME: feature flags depend on code loaded by the startup test. -->
```
Comment (makyo) [inline jujugui/static/gui/src/test/index.html]: Is this not the case anymore, then?

File: jujugui/static/gui/src/test/assets/chai.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/test/assets/chai.js]: lol - yeah these were symlinked so they needed to be updated for the new folder structure.

File: jujugui/static/gui/src/test/index.html (hunk 3 above)
Comment (hatched) [inline jujugui/static/gui/src/test/index.html]: This is still the case - I have created a card as a followup as we need to find a better way to do the testing of the js in the index file.

File: jujugui/static/gui/src/test/index.html (hunk 2 above)
Comment (hatched) [inline jujugui/static/gui/src/test/index.html]: ahh yeah leftover, I'll fix - needed it not combined to make debugging possible.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2755/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2756/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2758/
Test PASSed.

**Comment by makyo** [comment]
:+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2762/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2764/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2766/
Test PASSed.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**